tests/other/test_pthread_icu.cpp

```diff
@@ -0,0 +1,13 @@
+#include <thread>
+#include <iostream>
+#include <unicode/unistr.h>
+
+// Test that code using both pthread and icu compiles.
+int main () {
+	std::thread([] {
```

Comment:
I know we are not yet consistent, but we are trying to use a consistent style these days which involves 2-space indentation (we have a .clang-format file if you want to just use that)

Reply:
Sure. Can I recommend mentioning the existence of a style convention in [Contributing](https://emscripten.org/docs/contributing/contributing.html) or [Developer's Guide](https://emscripten.org/docs/contributing/developers_guide.html#developer-s-guide)?

Reply:
Yes we should do that.